resources/components/base/Message.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+	<div
+		class="mw-message"
+		v-bind:class="builtInClasses"
+		v-bind:aria-live="type !== 'error' ? 'polite' : false"
+		v-bind:role="type === 'error' ? 'alert' : false "
+	>
+		<icon
+			v-bind:icon="iconMap[ type ]"
+			v-bind:class="'oo-ui-image-' + type"
+		/>
+		<div class="mw-message__content">
+			<slot />
+		</div>
+	</div>
+</template>
+
+<script>
+var Icon = require( './Icon.vue' );
+
+module.exports = {
+	name: 'Message',
+
+	components: {
+		icon: Icon
+	},
+
+	props: {
+		type: {
+			type: String,
+			default: 'notice'
+		},
+		inline: {
+			type: Boolean,
+			default: false
+		}
+	},
+
+	data: function () {
+		return {
+			iconMap: {
+				notice: 'infoFilled',
+				error: 'error',
+				warning: 'alert',
+				success: 'check'
+			}
+		};
+	},
+
+	computed: {
+		typeClass() {
```

Comment:
Now that @catrope's linter rules have been merged, computed prop and method names should be written in ES5 syntax: `typeClass: function() {}` etc.